bcp/client/src/webpack/components/DataLayer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Maybe I'm missing something, but what's the point of this component?

Reply:
this is to a GeoJSON layer to show different over layers based on the json data feeded